be/src/exprs/map_functions.cpp

```diff
@@ -285,4 +285,74 @@ void MapFunctions::_filter_map_items(const MapColumn* src_column, const ColumnPt
     dest_column->values_column()->append_selective(src_column->values(), indexes);
 }
 
+StatusOr<ColumnPtr> MapFunctions::distinct_map_keys(FunctionContext* context, const Columns& columns) {
```

Comment:
`distinct_map` is more exact?

Reply:
not really, distinct_map should remove identical map with the same keys as well as values.